src/AutoRest.CSharp/Mgmt/Generation/ResourceOperationWriter.cs

```diff
@@ -158,7 +158,9 @@ private void WriteClientCtors(CodeWriter writer, ResourceOperation resourceOpera
         private void WriteClientProperties(CodeWriter writer, ResourceOperation resourceOperation, MgmtConfiguration config)
         {
             writer.Line();
+            writer.WriteXmlDocumentationSummary($"Gets the resource type for the operation group");
```

Comment:
Should it be called `operations` instead of `operation group` as in other comments we are using `operations`?

Reply:
Yes. Let's do this.